packages/message-input/theme/lumo/vaadin-message-input-styles.js

```diff
@@ -10,6 +10,10 @@ registerStyles(
     :host {
       padding: var(--lumo-space-s) var(--lumo-space-m);
     }
+
+    ::slotted([slot='textarea']) {
+      margin-inline-end: var(--lumo-space-s);
```

Comment:
Shouldn't `margin: 0` be added here like in the Material theme?

Reply:
It's not really needed, because Lumo version doesn't have any margin by default.